nav2_costmap_2d/include/nav2_costmap_2d/testing_helper.hpp

```diff
@@ -1,3 +1,17 @@
+// Copyright (c) 2019 Intel Corporation
```

Comment:
This was a port from Nav1 https://github.com/ros-planning/navigation/blob/melodic-devel/costmap_2d/include/costmap_2d/testing_helper.h

This should probably still have an original copyright header added if you're also going to add intels

Reply:
There is no copyright header in the original

Reply:
In that case, do keep the file copyright-less and suppress the lint fail?

@crdelsey do you have a suggestion on what to do in this case?

Reply:
Even without the copyright header, they still own the copyright itself. Adding the header without including the original copyright holder is basically claiming copyright over their work, which is pretty unambiguously infringing. The idea of a copyright header doesn't assert a copyright, its more of a "reminder" that isn't strictly required for any particular legal reason 

I'm just suggesting adding this up top as well 
` // Copyright (c) Willow Garage `

Reply:
But I approved it, not blocking over this